docs/repositories/software_eessi_io.md

```diff
@@ -0,0 +1,177 @@
+   EasyBuild/4.9.0                           (D)    Java/11.0.20                                (11)    nodejs/18.12.1-GCCcore-12.2.0                 SQLite/3.39.4-GCCcore-12.2.0
+   Eigen/3.4.0-GCCcore-12.3.0                       jbigkit/2.1-GCCcore-12.3.0                  (D)     nodejs/18.17.1-GCCcore-12.3.0          (D)    SQLite/3.42.0-GCCcore-12.3.0
+   expat/2.4.9-GCCcore-12.2.0                       JsonCpp/1.9.5-GCCcore-12.3.0                (D)     NSPR/4.35-GCCcore-12.2.0                      Szip/2.1.1-GCCcore-12.3.0                 (D)
+   expat/2.5.0-GCCcore-12.3.0                       LAME/3.100-GCCcore-12.3.0                   (D)     NSPR/4.35-GCCcore-12.3.0               (D)    Tcl/8.6.12-GCCcore-12.2.0
+   FFmpeg/6.0-GCCcore-12.3.0                 (D)    LHAPDF/6.5.4-GCC-12.3.0                             NSS/3.85-GCCcore-12.2.0                       Tcl/8.6.13-GCCcore-12.3.0
+   ffnvcodec/12.0.16.0                       (D)    libarchive/3.6.1-GCCcore-12.2.0                     NSS/3.89.1-GCCcore-12.3.0              (D)    TensorFlow/2.13.0-foss-2023a              (D)
+   FFTW.MPI/3.3.10-gompi-2022b                      libarchive/3.6.2-GCCcore-12.3.0                     nsync/1.26.0-GCCcore-12.3.0            (D)    Tk/8.6.13-GCCcore-12.3.0                  (D)
+   FFTW.MPI/3.3.10-gompi-2023a                      libdeflate/1.18-GCCcore-12.3.0              (D)     numactl/2.0.16-GCCcore-12.2.0                 UCC/1.1.0-GCCcore-12.2.0
+   FFTW/3.3.10-GCC-12.2.0                           libdrm/2.4.114-GCCcore-12.2.0                       numactl/2.0.16-GCCcore-12.3.0                 UCC/1.2.0-GCCcore-12.3.0
+   FFTW/3.3.10-GCC-12.3.0                           libdrm/2.4.115-GCCcore-12.3.0               (D)     OpenBLAS/0.3.21-GCC-12.2.0                    UCX/1.13.1-GCCcore-12.2.0
+   flatbuffers-python/23.5.26-GCCcore-12.3.0 (D)    libevent/2.1.12-GCCcore-12.2.0                      OpenBLAS/0.3.23-GCC-12.3.0                    UCX/1.14.1-GCCcore-12.3.0
+   flatbuffers/23.5.26-GCCcore-12.3.0        (D)    libevent/2.1.12-GCCcore-12.3.0                      OpenMPI/4.1.4-GCC-12.2.0                      UnZip/6.0-GCCcore-12.2.0
+   FlexiBLAS/3.2.1-GCC-12.2.0                       libfabric/1.16.1-GCCcore-12.2.0                     OpenMPI/4.1.5-GCC-12.3.0                      UnZip/6.0-GCCcore-12.3.0
+   FlexiBLAS/3.3.1-GCC-12.3.0                       libfabric/1.18.0-GCCcore-12.3.0                     OpenSSL/1.1                            (D)    virtualenv/20.23.1-GCCcore-12.3.0
+   flit/3.9.0-GCCcore-12.3.0                        libffi/3.4.4-GCCcore-12.2.0                         OSU-Micro-Benchmarks/7.1-1-gompi-2023a        VTK/9.3.0-foss-2023a                      (D)
+```
+
+### Architecture and micro-architecture support
+
+#### x86_64
+
+  - generic (currently implies `march=x86-64` and `-mtune=generic`)
```

Comment:
just point to https://www.eessi.io/docs/software_layer/cpu_targets/ (via `[...](../software_layer/cpu_targets.md)`)